code/src/utils/urls.js

```diff
@@ -0,0 +1,2 @@
+export const BASE_URL = `https://api.themoviedb.org/3/trending/movie/day?api_key=${process.env.REACT_APP_KEY}`
+export const SINGLE_MOVIE = (id) => `https://api.themoviedb.org/3/movie/${id}?api_key=${process.env.REACT_APP_KEY}`
```

Comment:
What is the "process.env.REACT..." doing? :) This is you API key, right? Where are the one million numbers etc that I always got as a key? 

Anyway, like that you got it in a seperate file - can imagine it's nice to have it this way in the future when we're doing larger project with a lot more components etc, easy to find.